packages/odyssey-react-mui/src/themes/odyssey/mixins.ts

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ThemeOptions } from "@mui/material";
+import * as Tokens from "@okta/odyssey-design-tokens";
+
+export const mixins: ThemeOptions["mixins"] = {
+  maxWidth: Tokens.FontLineLengthMax,
```

Comment:
@KevinGhadyani-Okta I had the same issue we saw previously when adding these to `Shape`. Instead of postponing this theming, I'd rather merge this, re-test adding theme variables separately, and refactor if needed.